NAS2D/Resources/Sprite.cpp

```diff
@@ -139,12 +139,12 @@ void Sprite::update(Point<float> position)
 		if (mCurrentFrame >= mActions[mCurrentAction].size())
 		{
 			mCurrentFrame = 0;
-			mFrameCallback(); // Notifiy any frame listeners that the animation sequence has completed.
+			mAnimationCompleteCallback(); // Notifiy any frame listeners that the animation sequence has completed.
 		}
 	}
 	else if (frame.frameDelay == FRAME_PAUSE)
 	{
-		mFrameCallback();
+		mAnimationCompleteCallback();
```

Comment:
You know, I'm thinking in a case like this the callback shouldn't get called. It's a single frame, why would we call this every update?

Reply:
You know, I was kind of wondering about that event trigger too. I suppose it was intended to trigger when it first hit a frame with a `frameDelay` of `FRAME_PAUSE`?